modules/guides/pages/javascript-udfs/remove-javascript-udf.adoc

```diff
@@ -0,0 +1,73 @@
+= Removing a JavaScript UDF
+:description: Removing a JavaScript UDF using either the UI, the REST API or the CBQ Shell.
+:page-pagination: 
+:page-edition: Enterprise Edition
+:page-topic-type: guide
+:page-toclevels: 2
+
+
+[abstract]
+{description}
+
+== Introduction
+
+You can remove a JavaScript UDF using the UDF screen on the Query Workbench. 
+The N1QL referencing function can be removed using the same tools.
+
+Removal of the JavaScript function is best carried out using the Query Workbench, especially if you have other functions contained in the same library.
+
+
+== Remove the JavaScript function
+
+. Edit the library that contains your function.
++
+image::javascript-udfs/removing-javascript-function.png[,600]
+
+. Delete your function from the library.
+. Press the btn:[Save] button.
+
+
+== Remove the N1QL Reference Function
+
+
+[tabs]
+====
+Query Workbench::
++
+--
+Access the `UDFs` screen from the Query Workbench.
+
+. Select btn:[Query] to access the Query Workbench, then select btn:[UDF] Query Workbench menu.
++
+image::javascript-udfs/navigate-to-udf-query.png[alt="route to the user-defined functions screen"]
+
+. Click on the `Drop` link that corresponds to the N1QL reference function for your JavaScript function.
++
+image::javascript-udfs/drop-sqlpp-handler.png[,600, alt="Dropping the N1QL handler function"]
+
+. Press the btn:[OK] button to confirm that you wish to delete the reference function.
+--
+
+REST API::
++
+--
+Start a command shell and execute the following command:
+
+[source, console]
+----
+include::example$javascript-udfs/drop-sqlpp-reference-function.sh[]
+----
```

Comment:
And conversely, we don't need an explicit section for this here. It's just the REST API call to execute a query, which is the same as the following section.